cloudant-sync-datastore-android/src/main/java/com/cloudant/sync/sqlite/android/AndroidSQLite.java

```diff
@@ -33,7 +33,9 @@
 
 
     public static AndroidSQLite createAndroidSQLite(String path) {
-        SQLiteDatabase db = SQLiteDatabase.openDatabase(path, null, SQLiteDatabase.CREATE_IF_NECESSARY);
+        SQLiteDatabase db = (path != null)
+                ? SQLiteDatabase.openDatabase(path, null, SQLiteDatabase.CREATE_IF_NECESSARY)
```

Comment:
Since `if/else` blocks are used in the rest of the code, wouldn't it be better to use that style to keep things consistent?